viewer/packages/sector-loader/tests/mockSectorUtils.ts

```diff
@@ -35,6 +35,6 @@ export function createWantedSectorMock(id: number = 1): IMock<WantedSector> {
 export function createBinaryFileProviderMock(): IMock<BinaryFileProvider> {
   const fileBuffer = fs.readFileSync(__dirname + '/test.glb');
   return new Mock<BinaryFileProvider>()
-    .setup(p => p.getBinaryFile(defaultBaseUrl, It.IsAny()))
+    .setup(p => p.getBinaryFile(defaultBaseUrl, It.IsAny(), undefined))
```

Comment:
Is this nessesary? I think if the function argument is optional, you don't need to pass undefined.

Reply:
I struggled a lot with this one. A lot of the tests fail (in weird places) if I do not pass something in here. But I will change it to It.IsAny(), instead of undefined. 

```
Test Suites: 2 failed, 95 passed, 97 total
Tests:       3 failed, 491 passed, 494 total
```